src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline

```diff
@@ -2549,6 +2549,73 @@ public class PublishSeriesRequestTestRequestObjectFieldComments {
 }
 // FIXME: Insert here clean-up code.
 
+// [END sample]
+============== file: samples/src/main/java/com/google/example/examples/library/v1/PublishSeriesRequestTestWriteToFile.java ==============
+// DO NOT EDIT! This is a generated sample ("Request",  "test_write_to_file")
+package com.google.example.examples.library.v1;
+
+import com.google.example.library.v1.Book;
+import com.google.example.library.v1.LibraryClient;
+import com.google.example.library.v1.PublishSeriesRequest;
+import com.google.example.library.v1.PublishSeriesResponse;
+import com.google.example.library.v1.SeriesUuid;
+import com.google.example.library.v1.Shelf;
+import java.io.FileOutputStream;
+import java.io.FileWriter;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+// [START sample]
+/*
+ * Please include the following imports to run this sample.
```

Comment:
nit: We should probably end this line with a colon for all samples.

Reply:
I'll do that in a separate PR to not pollute the baseline change.

Reply:
sg